### PR TITLE
(VANAGON-127) Prep for 0.15.6 release

### PR DESCRIPTION
**Pull request by branan**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,15 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
 
+## [0.15.6] - released on 2018-03-
+### Fixed
+- (VANAGON-125) On AIX, postinstall scripts for upgrades now run in
+  the triggerpostun phase. This ensure they run after any postun
```
Comment (mwaggett) [inline CHANGELOG.md]: 'ensure' should be 'ensures'

Comment (branan) [reply CHANGELOG.md]: You can't tell me how to grammar!

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,15 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
...
+  scripts which can interfere with the operation of the postinstall
+  scripts.
+- (VANAGON-125) On AIX, service stop in postun now only runs on an
+  uninstall, not on upgrades
```
Comment (mwaggett) [inline CHANGELOG.md]: end with period, please.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,15 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
 
+## [0.15.6] - released on 2018-03-
```
Comment (mwaggett) [inline CHANGELOG.md]: Just put today.

Comment (branan) [reply CHANGELOG.md]: ... I thought I did that

Comment (branan) [reply CHANGELOG.md]: I guess I need more :coffee:

**Comment by branan** [comment]
Fixed the ticket number

**Pull request merged**